JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/timeseries/TimeSeriesClient.java

```diff
@@ -172,13 +172,28 @@ else if(type.equals(Type.GENERAL)){
 	 * @param dataClass
 	 * @param timeUnit
 	 * @param conn
+	 * @param type
 	 * @param durations
 	 * @param units
      */
 	public void bulkInitTimeSeries(List<List<String>> dataIRIs, List<List<Class<?>>> dataClass, List<String> timeUnit, Connection conn, List<Type> type, List<Duration> durations, List<ChronoUnit> units) {
 		bulkInitTimeSeries(dataIRIs, dataClass, timeUnit, null, conn, type, durations, units);
 	}
 
+	/**
+	 * similar to initTimeSeries, but uploads triples in one connection
+	 * @param dataIRIs
+	 * @param dataClass
+	 * @param timeUnit
+	 * @param srid
+	 * @param conn
+	 * @param type
+	 * @param units
+	 */
+	public void bulkInitTimeSeries(List<List<String>> dataIRIs, List<List<Class<?>>> dataClass, List<String> timeUnit, Integer srid, Connection conn, List<Type> type, List<ChronoUnit> units) {
```

Comment:
In what circumstances do we need this combination? I thought Duration and ChronoUnit need to be provided as a pair

Reply:
this is a mistake, I'll rectify it.